<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpiderSetting</class>
 <widget class="QDialog" name="SpiderSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>353</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>353</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>爬虫设置</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>线程数目：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>爬行方向：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>爬行深度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>爬行宽度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>爬行数目：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>爬行时间(分)：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>关注</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>粉丝</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>点赞</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SpiderSetting</receiver>
   <slot>slotConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SpiderSetting</receiver>
   <slot>slotReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotConfirm()</slot>
  <slot>slotReset()</slot>
 </slots>
</ui>
